unity-client/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/SceneAssetPack.cs

```diff
@@ -15,9 +16,9 @@ public class SceneAssetPack
     public string updated_at;
     public string eth_address;
     public List<SceneObject> assets;
+   
+    string baseUrl = BuilderInWorldSettings.BASE_URL_ASSETS_PACK_CONTENT;
```

Comment:
Why do we need this wrapper around a constant? we could just use the constant

Reply:
Brian said that we should migrate the URLs to a config file just in case we need to migrate the API, it will be easy to follow the URLS